tests/tui/test_basic.py

```diff
@@ -0,0 +1,144 @@
+            await pilot.click("#time_slicing_switch")
+
+            # Check meta data modal. Click No to 'Proceed with these values?'
+            # await pilot.click(offset=(116, 31))
+            await pilot.click("#only_one_button")
+            # Specify slice acquisition direction, choose second choice
+            # await pilot.click(offset=(65, 26))
+            await pilot.click("#set_value_modal")
+            await pilot.press("down")
+            await pilot.press("enter")
+            # Click ok
+            # await pilot.click(offset=(132, 30))
+            # Click ok on the warning modal: Missing images
+            await pilot.click(pilot.app.get_widget_by_id("only_one_button"))
+
+            # await pilot.click(offset=(116, 31))
+            # await pilot.click("#only_one_button")
+            # Specify Slice timing modal: Choose third options
+            # await pilot.click(offset=(65, 25))
+            await pilot.click("#radio_set")
+            for _i in range(2):
+                await pilot.press("down")
+            await pilot.press("enter")
+
+            # Click ok on the 'Specify Slice timing modal'
+            # await pilot.click(offset=(132, 34))
+            await pilot.click("#ok")
+
+            # # click on 'pen' button to edit number of initial volumes to remove
+            # await pilot.click(offset=(121, 31))
+            await pilot.click("#edit_vols_to_remove_button")
+            # # click in the input box
+            # await pilot.click(offset=(101, 26))
+            await pilot.click("#input_prompt")
+            # Type '9'
+            await pilot.press("9")
+            # Click on Ok to confirm
+            # await pilot.click(offset=(101, 30))
+            await pilot.click("#ok")
+            # Select Check and Run tab
+            await pilot.press("r")
+            # Click 'Refresh'
+            # await pilot.click(offset=(85, 9))
+            await pilot.click("#refresh_button")
+            await settable_scroll_screen_down(pilot, 10)
+
+
+def test_work_dir_tab(snap_compare, start_app, work_dir_path: Path, downloaded_data_path: Path) -> None:
+    """Check whether one can set the working directory."""
+    run_before_with_extra_args = partial(
+        run_before, data_path=downloaded_data_path, work_dir_path=work_dir_path, stage="work_tab"
+    )
+    assert snap_compare(app=start_app, terminal_size=(204, 53), run_before=run_before_with_extra_args)
+
+
+def test_bids_data_input_tab(snap_compare, start_app, work_dir_path: Path, downloaded_data_path: Path) -> None:
+    """Check whether a bids data can be loaded. This should yield some non-zero found files at the file summary panel."""
+    run_before_with_extra_args = partial(
+        run_before, data_path=downloaded_data_path, work_dir_path=work_dir_path, stage="bids_data_tab"
+    )
+    assert snap_compare(app=start_app, terminal_size=(204, 53), run_before=run_before_with_extra_args)
+
+
+def test_non_bids_data_input_tab(
```

Comment:
A lot of these tests are very similar. Would it make sense to merge some of them and parametrize them as a fixture? That would reduce code duplication and make maintenance easier